src/ocean/io/select/client/SelectEvent.d

```diff
@@ -87,11 +87,25 @@ public abstract class ISelectEvent : IAdvancedSelectClient
         Constructor. Creates a custom event and hooks it up to the provided
         event handler.
 
+        Supported OR-combination of `flags`, starting with Linux 2.6.27:
+          - `EFD.NONBLOCK`: Set the `O_NONBLOCK` file status flag on the new
```

Comment:
It is true that this is a separate issue with no demand. Still I think it's bad design to not make these parameters available. But this should probably better be done in a separate PR.